### PR TITLE
[Course Task] doc-device-crd

### PR DESCRIPTION
**Pull request by jxs1211**
Signed-off-by: xian-jie.shen <327411586@qq.com>

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
docs update

* **What is the current behavior?** (You can also link to an open issue here)
[129](https://github.com/kubeedge/community/issues/129)

* **What is the new behavior (if this is a feature change)?**
none

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
none

* **Other information**:
none

### CODE REVIEW

File: content/en/docs/developer/device_crd.md [hunk 1]
```diff
@@ -205,22 +202,17 @@ The following are the steps to
     ```
 
 2. Create a device instance in the cloud node.
-
-    ```shell
-           kubectl apply -f <path to device instance yaml>
-    ```
-
```
Comment (kevin-wangzefeng) [inline content/en/docs/developer/device_crd.md:212]: I think we shall keep these lines?

Comment (jxs1211) [reply content/en/docs/developer/device_crd.md]: updated

**Review by kevin-wangzefeng — COMMENTED** [review]
/lgtm
/approve

/hold for mannual merging

**Pull request merged**